pygfunction/api.py

```diff
@@ -0,0 +1,39 @@
+from pygfunction.gfunction import gFunction
+from pygfunction.boreholes import Borehole
+from typing import Dict, Optional, Any, List
```

Comment:
Just as a quick note, Python's built in `list` and `dict` types got support for type hinting, so you shouldn't need to use the upper cased typing classes anymore.

Reply:
@Myoldmopar Since Massimo is supporting Python 3.8 we gotta do the old way. I know, using modern Python would be nice. If PyGFunction goes to a minimum of 3.9 (or preferably 3.10) we can change hinting: https://www.perplexity.ai/search/what-version-of-python-got-bui-SZAjYcxqQyqeux3FI2uilQ#0

Reply:
Gah, that makes sense.  With 3.8 now past EOL, I assume this won't be a problem for much longer.